themes/startwords/assets/scss/issue/_summary.scss

```diff
@@ -38,6 +38,10 @@
         margin-left: 0;
         font-weight: 300;
     }
+    
+    .pubdate {
+        font-size: rem(16px);
```

Comment:
Previously, the issue list page inherited a font weight of 400 for the issue publication date. This change actually lightened it to 300, which was difficult to read between the two chunks of 700 and 900 text.